### PR TITLE
Refactor the CLI, part 3

### PR DESCRIPTION
**Pull request by fghaas**
* Increase test coverage
* Put Git functionality into a separate helper class
* Update README and convert to reStructuredText, so it can be included in `setup.py` as the package's `long_description`.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ def package_data(pkg, root_list):
     name='olx-utils',
     version='0.1.1',
     description='Utilities for edX OLX courses',
+    long_description=open('README.rst').read(),
```
Comment (arbrandes) [inline setup.py]: RST kinda sucks compared to markdown, but this is a very nice way to avoid redundancy, indeed.

Comment (fghaas) [reply setup.py]: You can always write whatever you like in Markdown and then use pandoc (i.e. legit and true magic) to convert to RST.

**Comment by fghaas** [comment]
Alrighty, then we went from 30% to 90+% test coverage in a week. Doesn't suck. :)

**Comment by fghaas** [comment]
Thanks for reviewing!

**Pull request merged**